tests/support/CodeHelper.php

```diff
@@ -1,7 +1,8 @@
 <?php
+
 namespace Codeception\Module;
 
-// here you can define custom functions for CodeGuy 
+// here you can define custom functions for CodeGuy
```

Comment:
I will delete all `define custom functions` and `Define custom actions` comments in tests/ too.